include/pru/am335x.h

```diff
@@ -26,6 +26,11 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* 
+* Updated according to AM335x TRM Rev3
```

Comment:
Please write the full document number, including the revision. For rationale see next comment bellow.

Reply:
Fixed in f70e9cc5cdc9400b0b5be2668915ba2d55c20cb8